package.json

```diff
@@ -105,7 +105,7 @@
     "@fastify/busboy": "2.1.1",
     "@matteo.collina/tspl": "^0.1.1",
     "@sinonjs/fake-timers": "^11.1.0",
-    "@types/node": "^18.0.3",
+    "@types/node": "~18.17.19",
```

Comment:
why?

Reply:
This just provides type for node 18.17 specifically, which is the minimum required based on the engines version. The listed version `^18.0.3` doesn't contain the `Event` and `EventTarget`. And the version that ends up being installed installed typing related to node 18.19 (18.19.47) instead of the minimum supported 18.17

Reply:
I can revert back this change if necessary, but I strongly recommend locking the `@types/node` to the minimum supported version mentioned in `engines.node` in `package.json`.